src/Umbraco.Web.UI.Client/src/common/services/navigation.service.js

```diff
@@ -118,19 +118,25 @@ function navigationService($routeParams, $location, $q, $injector, eventsService
     }
 
     function closeBackdrop() {
-        var aboveClass = 'above-backdrop';
-        var leftColumn = $('#leftcolumn');
-        var isLeftColumnOnTop = leftColumn.hasClass(aboveClass);
 
-        if(isLeftColumnOnTop){
+        var tourIsOpen = document.body.classList.contains("umb-tour-is-visible");
+        if (tourIsOpen) {
+            return;
+        }
+
+        var aboveClass = "above-backdrop";
+        var leftColumn = document.getElementById("leftcolumn");
```

Comment:
Apparently this could result in a null result, so the tests were failing, fixed in https://github.com/umbraco/Umbraco-CMS/commit/d55b740b16faf8b8faaffa533ee5130371ae6169

Reply:
In that case `if (leftColumn)` should be sufficient :)